xarray/datatree_/datatree/tests/conftest.py

```diff
@@ -0,0 +1,65 @@
+import pytest
```

Comment:
this likely needs to migrate to `xarray/tests/conftest.py`

Reply:
this one might be OK! conftest does work hierarchically! To the extent the tests are staying here...

Reply:
`xarray/datatree_` is a temporary directory containing the full import. We chose this strategy so that we can use PRs to move every file / piece of code to the right place (and after the migration is done we will remove it)